index.html

```diff
@@ -3,8 +3,39 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Hello Microverse!</h1>
+        <header class="header">
+            <a class="logo">My Logo</a>
+            <nav><img class="menu" src="Union.png" alt="MenuBar"></nav>
+        </header>
+        <section class="notheader">
+            <div class="textcontainer">
+                <h1 class="primarytext">Hey There.<br>
+                    I'm Gabriel.
+                </h1>
+                <h2 class="supportingtext">
+                    I'm a Software Developer
+                </h2>
+                <p class="description">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </p>
+            </div>
+            <ul class="socialmedia">
+                <li><a><img class="mediaicon" src="Disabled.png" alt="SociaMediaIcon"></a></li>
+                <li><a><img class="mediaicon" src="Disabled-1.png" alt="SociaMediaIcon"></a></li>            
+                <li><a><img class="mediaicon" src="Enabled.png" alt="SociaMediaIcon"></a></li>
+                <li><a><img class="mediaicon" src="Disabled-2.png" alt="SociaMediaIcon"></a></li>
+                <li><a><img class="mediaicon" src="Disabled-3.png" alt="SociaMediaIcon"></a></li>
```

Comment:
[OPTIONAL]

- KIndly remember to add` href `attribute to your `<a>` tag for your website accessibility 👍🏽